plugins/module_utils/_global_catalog.py

```diff
@@ -6,14 +6,14 @@
 # FOR INTERNAL USE IN THE COLLECTION ONLY.
 
 from __future__ import (absolute_import, division, print_function)
+import os
```

Comment:
Can we not do open imports like this. We only use `os.remove` so dont need to import the entirety of os. Same goes for tempfile below